warehouse/templates/packaging/detail.html

```diff
@@ -147,14 +149,39 @@ <h1 class="package-header__name">
       </h1>
 
       {% if files %}
-      <p class="package-header__pip-instructions">
-        <span id="pip-command">{{ pip_command(release, request.matched_route.name, testPyPI) }}</span>
-        <button type="button" class="copy-tooltip copy-tooltip-s" data-clipboard-target="#pip-command" data-tooltip-label="{% trans %}Copy to clipboard{% endtrans %}">
-          <i class="fa fa-copy" aria-hidden="true"></i>
-          <span class="sr-only">{% trans %}Copy PIP instructions{% endtrans %}</span>
-        </button>
-      </p>
-      {% endif %}
+      <div
+        data-controller="package-manager-instructions"
+        data-package-manager-instructions-package-name-value="{{ release.project.name }}"
+        data-package-manager-instructions-project-version-value="{{ project_version }}"
+        data-package-manager-instructions-index-url-value="{{ index_url }}"
+      >
+        <p class="package-header__pip-instructions">
+          <span id="pip-command" data-package-manager-instructions-target="command">{{ pip_command(release, request.matched_route.name, testPyPI) }}</span>
```

Comment:
The `pip_command` macro is located above and contains this snippet:
https://github.com/pypi/warehouse/blob/3b4a3113dcd0ad438c2ceebda3c274351561042f/warehouse/templates/packaging/detail.html#L85-L87

This has the affect of transforming the copyable command from

<img width="464" alt="Screenshot 2023-01-08 at 5 28 29 PM" src="https://user-images.githubusercontent.com/529516/211222293-c04a6758-2f7f-4db8-a008-f6128307852d.png">

to:

<img width="520" alt="Screenshot 2023-01-08 at 5 28 48 PM" src="https://user-images.githubusercontent.com/529516/211222310-f87e8c0f-cbfe-4e37-9c87-2e873868ef60.png">

See https://pypi.org/project/hellowebapp-deploy/ vs https://pypi.org/project/hellowebapp-deploy/1.0.5/

By changing `innerHTML`, the existing behavior is broken after a change away from `pip` and back.

It'd be great map out the correct syntax for each package manager and version specifics, but at the least we shouldn't break current behavior.